runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -663,10 +663,25 @@ TR_VectorAPIExpansion::getJ9ClassFromClassNode(TR::Compilation *comp, TR::Node *
          {
          TR_J9VMBase *fej9 = comp->fej9();
 
-         TR::VMAccessCriticalSection getDataTypeFromClassNodeSection(fej9);
+      J9Class *j9class = NULL;
+#if defined(J9VM_OPT_JITSERVER)
+      if (comp->isOutOfProcessCompilation()) // In server mode
+         {
+            auto stream = TR::CompilationInfo::getStream();
```

Comment:
Get the stream from the compilation object because it is cheaper than accessing a thread local variable.